python/cudf/cudf/tests/test_cuda_apply.py

```diff
@@ -83,8 +83,7 @@ def test_df_apply_custom_chunks(nelem):
     def kernel(in1, in2, in3, out1, out2, extra1, extra2):
         for i, (x, y, z) in enumerate(zip(in1, in2, in3)):
             out1[i] = extra2 * x - extra1 * y + z
-            # cuda.blockDim.x is 1
-            out2[i] = i * cuda.blockDim.x
+            out2[i] = i
```

Comment:
Why drop `cuda.blockDim.x` here?

Reply:
Because that was specifically written assuming the `tpb` setting was `1` which it no longer is.